src/vuestic-theme/vuestic-components/va-input/VaInputWrapper.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div
+    class="va-input-wrapper d-flex align--end text--left"
+    :class="{ 'va-input-wrapper--disabled' : disabled }"
+  >
+    <div class="va-input-wrapper__control">
+      <div
+        tabindex="0"
+        :style="slotStyles"
+        class="va-input-wrapper__slot d-flex align--end pt-2">
+        <div
+          v-if="hasPrependData"
+          class="va-input-wrapper__prepend-inner d-inline-flex align--center ml-2">
+          <slot name="prepend"/>
+        </div>
+        <div class="va-input-wrapper__content">
+          <slot/>
+        </div>
+        <div
+          v-if="hasAppendData"
+          class="va-input-wrapper__append-inner d-inline-flex align--center mr-2">
+          <slot name="append"/>
+        </div>
+      </div>
+      <div class="va-input-wrapper__details py-0 px-2">
+        <div class="va-input-wrapper__messages">
+          <div
+            v-if="error"
+            :style="messageStyles"
+            class="va-input-wrapper__messages__wrapper">
+            <template
+              v-for="message in errorMessages">
+              {{ message }}
```

Comment:
You have `{{ message }}` in the template 8 lines below. Could be confusing. I suggest `v-for="errorMessage in errorMessages"`